### ISSUE TITLE
Custom eMail adresses for teachers / extrastudents

### ISSUE THREAD

**Issue by zefanja** [linuxmuster/sophomorix4#125]
Thanks for fixing #115, but I have another issue.

Many of our staff have email addresses related to their role / position in school e.g. principle@exmaple.com or secretary@example.com. Is there a way to solve this in sophomorix?

I can directly edit their AD entry, but the next time I run sophomorix-update it is changed to the old value.

Another workaround is to add an alias in our mail server config for all this addresses (that's how we do it now).

**Comment by dorianim** [comment]
The correct solution for this would be to use the attribute "proxyAddress".
But I don't think there currently is a way to do this with sophomorix.

**Comment by kiarn** [comment]
Yes, as far as I know it's `proxyAddress` or `sophomorixCustom`.

To use `proxyAddress` : 

```
# Updating the user configurable E-Mail address attribute proxyAddresses (multi-value):
  --user <user> --set-proxy-addresses <address1>,<address2>,...
  --user <user> --add-proxy-addresses <address>
  --user <user> --remove-proxy-addresses <address>
sophomorix-user -u USER --set-proxy-addresses "test@gmail.com"
```

To use custom : 

```
sophomorix-user -u USER --custom1 "test@gmail.com"
```

Arnaud

**Comment by dorianim** [comment]
I didnt know that sophomorix allows to actually change the proxyAddress. I will open an issue in the webui regarding this :)

**Comment by kiarn** [comment]
Not necessary, we can discuss this tomorrow, we can couple this with the new custom handle.

**Comment by dorianim** [comment]
Too late 🙃
But it will be there as a reminder so it wont get lost.
